chains/canto/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://explorer.plexnode.wtf/api
+    family: blockscout
+    name: Canto Explorer
+    url: https://explorer.plexnode.wtf/
+chainId: 7700
+deployer:
+  name: guy93354
+  url: https://github.com/guy93354
+displayName: Canto
+domainId: 7700
+isTestnet: false
+name: canto
+nativeToken:
+  decimals: 18
+  name: Canto
+  symbol: CANTO
+protocol: ethereum
+rpcUrls:
+  - http: https://canto-rpc.ansybl.io
+  - http: https://mainnode.plexnode.org:8545
+  - http: https://canto.slingshot.finance
```

Comment:
Hey @guy93354 thank you for your PR to the Hyperlane Registry! Can you run `yarn prettify` to fix this formatting issue and update the branch?

Reply:
Hey @xeno097, thanks for you feedback, I didn't notice this formatting error.
I ran `yarn pretiffy` and uploaded the newest metadata.yaml file